src/js/data.js

```diff
@@ -2,43 +2,74 @@ const serverHostname = 'localhost';
 const serverPort = 8080;
 const timelinePath = 'api/1.0/twitter/timeline';
 
+const month = new Array();
+month[0] = "Jan";
+month[1] = "Feb";
+month[2] = "Mar";
+month[3] = "Apr";
+month[4] = "May";
+month[5] = "Jun";
+month[6] = "Jul";
+month[7] = "Aug";
+month[8] = "Sep";
+month[9] = "Oct";
+month[10] = "Nov";
+month[11] = "Dec";
+
 function getTimeline() {
 	let xhttp = new XMLHttpRequest();
 
     xhttp.onreadystatechange = function () {
     	if (xhttp.readyState == xhttp.DONE) {
         	if (xhttp.status == 200) {
-                document.getElementById('tweetTable').innerHTML = "";
+                document.getElementsByClassName("tweetTable").innerHTML = "";
                 handleData(xhttp.responseText);
         	}
         	else {
-        		document.getElementById('tweetTable').innerHTML = "An error has occurred. Please contact your administration.";
+                errorMessage();
         	}
         }
         else {
-	        document.getElementById('tweetTable').innerHTML = "Retrieving information, please wait!";
+            waitMessage();
         }
     }
 
     xhttp.open("GET", `http://${serverHostname}:${serverPort}/${timelinePath}`, true);
     xhttp.send();
 }
 
+function errorMessage() {
+    let tweetTable = document.getElementsByClassName("tweetTable")[0];
+    tweetTable.innerHTML = "";
+
+    let error = document.createElement("div");
+    error.setAttribute("class", "error");
+    error.innerHTML = "An error has occurred. Please contact your administration.";
+
+    tweetTable.appendChild(error);
+}
+
+function waitMessage() {
+    let tweetTable = document.getElementsByClassName("tweetTable")[0];
+    tweetTable.innerHTML = "";
+
+    let wait = document.createElement("div");
+    wait.setAttribute("class", "wait");
+    wait.innerHTML = "Retrieving information, please wait!";
+
+    tweetTable.appendChild(wait);
+}
+
 function handleData(data) {
+    console.log("Handling data");
```

Comment:
if you don't need the console.log, please clean up

Reply:
Done